### PR TITLE
Return error on appVersion if package not found

### PR DESCRIPTION
**Pull request by RalfNieuwenhuizen**
## Description

Instead of the default `not-available`, return `error` when a package can not be found (which should not happen). Better for debugging purposes.

## Compatibility

| OS      | Implemented |
| ------- | :---------: |
| iOS     |    ❌     |
| Android |    ✅     |
| Windows |    ❌     |

## Checklist

* [X] I have tested this on a device/simulator for each compatible OS
* [ ] I added the documentation in `README.md`.
* [ ] I mentionned this change in `CHANGELOG.md`.

### CODE REVIEW

File: android/src/main/java/com/learnium/RNDeviceInfo/RNDeviceModule.java [hunk 1]
```diff
@@ -115,7 +115,6 @@ public void getMacAddress(Promise p) {
     String packageName = this.reactContext.getPackageName();
 
     constants.put("appVersion", "not available");
-    constants.put("buildVersion", "not available");
```
Comment (machour) [inline android/src/main/java/com/learnium/RNDeviceInfo/RNDeviceModule.java]: Why was this line removed?

File: android/src/main/java/com/learnium/RNDeviceInfo/RNDeviceModule.java [hunk 2]
```diff
@@ -125,6 +124,7 @@ public void getMacAddress(Promise p) {
       constants.put("firstInstallTime", info.firstInstallTime);
       constants.put("lastUpdateTime", info.lastUpdateTime);
     } catch (PackageManager.NameNotFoundException e) {
+      constants.put("appVersion", "error");
```
Comment (machour) [inline android/src/main/java/com/learnium/RNDeviceInfo/RNDeviceModule.java]: This sadly causes a BC break for people relying on the string "not available" being returned upon error :/

I'll try to introduce a nicer debug mode for 1.0.0 (#307)

**Comment by RalfNieuwenhuizen** [comment]
@machour you're probably right, the fix is too specific for my use case :)

**Comment by machour** [comment]
@RalfNieuwenhuizen no problems mate :)

If you have any suggestions about how error reporting/hinting could be enhanced for the next BC breaking release, I'm all ears!
